net_adapter_linux.c

```diff
@@ -10,7 +10,12 @@
 #include <unistd.h>
 #include <ifaddrs.h>
 #include <net/if.h>
-#include <net/if_arp.h>
+#ifdef HAVE_NET_IF_ARP_H
+#  include <net/if_arp.h>
```

Comment:
3-space indent?

Reply:
It is only two space indent. It is part of my lint configuration.